_posts/2022-09-05-edge-example-app-cpu-temp.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Edge Example App: CPU Temperature"
+date:   2022-09-05 03:27:05 +0100
```

Comment:
This is another blog post, I'd like to suggest to open a separate PR

Reply:
I made a PR [here](https://github.com/project-flotta/project-flotta.github.io/pull/113)

Reply:
Thanks, please remove it from here

Reply:
done